UploadDaemon_Test/SymbolAnalysis/SymbolCollectionResolverTest.cs

```diff
@@ -33,19 +33,44 @@ public void TearDown()
         }
 
         [Test]
-        public void ConsidersSymbolFileIncludes()
+        public void ConsidersSymbolFileIncludesFromSymbolDirectory()
         {
             SymbolCollection collection = resolver.ResolveFromSymbolDirectory(TestSymbolDirectory,
                    new GlobPatternList(new List<string> { "DoesNotExist*" }, new List<string> {}));
 
             Assert.That(collection.IsEmpty, Is.True);
-        }
 
-        [Test]
+			collection = resolver.ResolveFromSymbolDirectory(TestSymbolDirectory,
```

Comment:
Tabs vs spaces for indentation? There seem to be tabs.

Reply:
The formatter seems to switch to tabs, weird 😅

Reply:
Found the setting, now it should be correct again :grin: